app/Console/Commands/ImportFragments.php

```diff
@@ -45,12 +45,23 @@ private function importFragmentGroup(string $group, array $fragments)
                 $fragment->setTranslation($locale, $text ?: '');
             }
 
-            $fragment->html = $attributes['html'] ?? false;
+            $fragment->html = $attributes['html'] ?? $this->translationsContainHtml($translations);
             $fragment->description = $attributes['description'] ?? null;
 
             $fragment->save();
 
             return $fragment;
         })->values();
     }
+
+    private function translationsContainHtml(array $translations): bool
+    {
+        foreach ($translations as $translation) {
+            if ($translation != strip_tags($translation)) {
```

Comment:
Let's use a strict comparison here.

Reply:
I didn't use a strict comparison here because an empty fragment translation is `null` but using `strip_tags` an empty string is returned... Same for fragments that only contain a number (`company.postal`).